packages/react-components/src/components/RichTextEditor/Toolbar/RichTextToolbar.tsx

```diff
@@ -110,14 +110,23 @@ export const RichTextToolbar = (props: RichTextToolbarProps): JSX.Element => {
       icon: 'RichTextBulletListButtonIcon',
       onClick: () => {
         plugin.onToolbarButtonClick((editor) => {
+          // check the format state to see if the bulleted list is already applied
+          const isBullet = formatState?.isBullet;
           toggleBullet(editor);
+          // the bulleted list was added
+          if (!isBullet) {
+            setTimeout(() => {
+              // a small delay and polite aria live are needed for MacOS VoiceOver to announce the change
```

Comment:
Is this delay something we have done for other announcements?

Reply:
THere are some places. In this case we change focus when toggle list functions are called, adding the delay helps MacOS to queue everything correctly

By the way, Windows works without it as per RoosterJS engineer feedback

Reply:
also, the delay value is 50 milliseconds which is pretty small amount